src/sources/metadatasource.cpp

```diff
@@ -11,15 +11,17 @@ const Logger kLogger("MetadataSource");
 } // anonymous namespace
 
 // static
-QDateTime MetadataSource::getFileSynchronizedAt(const QFileInfo& fileInfo) {
-    const QDateTime lastModifiedUtc = fileInfo.lastModified().toUTC();
+QDateTime MetadataSource::getFileSynchronizedAt(const QFile& file) {
```

Comment:
This does not return a "Synchronized" time, because it can have various reason that the FileModificationTime has changed. The word "Synchronized" implies a second instance to synchronize with was there. How about:
  ```suggestion
QDateTime MetadataSource::getFileMofificationTime(const QFile& file) {
```

Reply:
I disagree. This function has been introduced do break the dependency on raw file modification time stamps. The returned value is not the actual file modification time stamp.

Reply:
Sorry, but no.

I have commented **every single line** of code in the implementation. The API comments in the header file should not reveal implementation details.

Reply:
Please make an example what could be an alternative implementation. 

This issue here is that the current function name implies that Mixxx has synchronized the file. But this wit wrong. 
Conclusion: The function name is wrong.

Reply:
I disagree and refuse to make any changes.

Reply:
In any case the method name was not changed in the PR and should not block merge. @daschuer Feel free to propose another name in a follow-up PR if you consider it necessary.

Reply:
OK